executor/executor_test.go

```diff
@@ -185,6 +185,20 @@ func (s *testSuite) TestAdmin(c *C) {
 	c.Assert(row.GetString(1), Equals, historyJobs[0].State.String())
 	c.Assert(err, IsNil)
 
+	// show DDL job queries test
+	s.TearDownSuite(c)
+	s.SetUpSuite(c)
```

Comment:
We'd better not use `TearDownSuite` and `SetUpSuite` in a specific test.
Rest LGTM